Snippets/Snippets_3/DataBus/CleanupStrategy/Usage.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Snippets3.DataBus.CleanupStrategy
+{
+    using System;
+    using System.IO;
+    using Snippets3.DataBus.DataBusProperty;
+
+    public class Usage
+    {
+        #region FileLocationForDatabusFiles
+        public void Handle(MessageWithLargePayload message)
+        {
+            string filename = Path.Combine(@"c:\databus_files\", message.LargeBlob.Key);
```

Comment:
@dvdstelt - sorry i missed your question. I was thinking more of a simple timer that executes a cleanup operation/check. @SimonCropp - thoughts?